docs/csharp/quick-starts/introduction-to-classes.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Quick Starts - Introduction to Classes - C# Guide
+description: Create your first C# program and explore object oriented concepts
+keywords: C#, Get Started, Lessons, Classes, Object Oriented Programming
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/11/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# Introduction to classes
+
+This lesson assumes installed [.NET Core](http://dot.net/core) and [Visual Studio Code](https://code.visualstudio.com/), or [Visual Studio](https://www.visualstudio.com/) on Mac or Windows.
+
+## Create your application
+
+Create a directory named **classes**. You'll build your application there. Change to that directory and type `dotnet new console`. This command creates your application. Open **Program.cs**. It should look like this:
+
+```csharp
+using System;
+
+namespace classes
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+In this quick start, you're going to create new types that represent a bank account. Typically developers define each class in a different text file. That makes it easier to manage as a program grows in size.  Create a new file named **BankAccount.cs** in the **classes** directory. 
+
+This file will contain the deefinition of a ***bank account***. Object Oriented programming organizes code by creating types in the form of ***classes***. These classes contain the code that represent a concept. The `BankAccount` class represents the responsibilities and properties of a bank account. In this quick start, the bank account supports this behavior:
+
+1. It has a 10-digit number that uniquely identifies the bank account.
+1. It has a string that stores the name or names of the owners.
+1. The balance can be retrieved.
+1. It accepts deposits.
+1. It accepts withdrawals.
```

Comment:
Do you want to add any additional rule, such as a withdrawal must not exceed the balance?